tests/conftest.py

```diff
@@ -132,13 +132,25 @@ def full_build_plan(mocker) -> list[models.BuildInfo]:
     return build_plan
 
 
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture()
```

An autoformat (`tox -f format`) should take care of that too